### PR TITLE
Release workflow

### PR DESCRIPTION
**Pull request by samuelattwood**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+  # version updates: disabled
+  # security updates: enabled
+  # https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates#overriding-the-default-behavior-with-a-configuration-file
+  # ./go
+  - package-ecosystem: gomod
+    directory: /
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 0
```
Comment (ripienaar) [inline .github/dependabot.yml:16]: generally find dependabot just annoying and noisy, I tend to update all dependencies almost weekly on main, and for release branches we would anyway do it right before release.

**Review by ripienaar — APPROVED** [review]
LGTM, of course who knows with github actions.

Small comment on dependabot I just really dont want the daily annoyance

File: .github/dependabot.yml (hunk 1 above)
Comment (samuelattwood) [reply .github/dependabot.yml:16]: I can remove it if you prefer. The config I added will only open go module PRs for security updates, not for general updates.

Comment (ripienaar) [reply .github/dependabot.yml:16]: Yeah if you dont mind, that'd be good

Comment (samuelattwood) [reply .github/dependabot.yml]: I left the weekly check for github actions updates but removed the golang config. The former should be very infrequent and is easier to lose track of.

**Pull request merged**